api/Makefile.am

```diff
@@ -45,7 +45,7 @@ endif
 libtesseract_api_la_SOURCES = baseapi.cpp capi.cpp renderer.cpp pdfrenderer.cpp
 
 lib_LTLIBRARIES += libtesseract.la
-libtesseract_la_LDFLAGS = 
+libtesseract_la_LDFLAGS = $(OPENCL_LDFLAGS)
```

Comment:
Yes, my initial guess was correct in this case. By `local_LDFLAGS` I meant there is a local variable somewhere with the suffix `_LDFLAGS` that was not being set correctly.